test/integration/connect/envoy/case-cfg-router-features/verify.bats

```diff
@@ -56,6 +56,10 @@ load helpers
   assert_expected_fortio_name s2-v1 localhost 5000 /prefix-alt
 }
 
+@test "test prefix path case insensitive" {
+  assert_expected_fortio_name s2-v1 localhost 5000 /prefix-case-Insensitive
```

Comment:
Should we perhaps make a few more calls to prove that the case is ignored for this?

```
assert_expected_fortio_name s2-v1 localhost 5000 /prefix-case-INSENSITIVE
assert_expected_fortio_name s2-v1 localhost 5000 /prefix-case-insensitive
```

Reply:
Here it is :-) https://github.com/hashicorp/consul/pull/19647/commits/5c7a2a0138a7472716c2a27765790ec1f5e173ee